<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QDialog#Dialog{
background: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(170, 170, 255, 255), stop:1 rgba(255, 170, 255, 255))
}</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="file_box"/>
    </item>
    <item>
     <widget class="QPushButton" name="file_button">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 255);</string>
      </property>
      <property name="text">
       <string>Select File</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSpinBox" name="start_row_box">
      <property name="maximum">
       <number>99999</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="end_row_box">
      <property name="maximum">
       <number>99999</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="add_range_button">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 255);</string>
      </property>
      <property name="text">
       <string>Add Range</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>231</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>start</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>end</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Select Sheet</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="sheet_box">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 127);
background-color: rgb(85, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="start_button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#start_button{
	background: rgb(85, 255, 127);
	border-radius: 10px;
	border: 2px solid rgb(0, 170, 127);
	border-width: 1px;
}


QPushButton#start_button:hover{
	background: rgba(85, 255, 127, 200);
}

QPushButton#start_button:pressed{
	background: rgba(180, 255, 233, 200);
}

rgb()</string>
   </property>
   <property name="text">
    <string>Start Morbin</string>
   </property>
  </widget>
  <widget class="QLabel" name="console">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>410</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
font-size: 16px;
color: rgb(255, 0, 127)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>95</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 85, 127);
	border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_box">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>New AP Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mac_box">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Formatted AP Mac</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="existing_name_box">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Existing AP name (optional)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
